src/server/scripts/EasternKingdoms/Deadmines/instance_deadmines.cpp

```diff
@@ -87,13 +95,17 @@ class instance_deadmines : public InstanceMapScript
                 if (!pIronCladDoor)
                     return;
 
+                // Loads Mr. Smite's grid. We need this as he must be active in order for his texts to work.
+                instance->LoadGrid(MrSmite.GetPositionX(), MrSmite.GetPositionY());
```

Comment:
This should be called on the same event his text is. The placement of this here will cause it to be executed on every update (unnecessary overload)